stylus/utilities/text.styl

```diff
@@ -174,11 +174,29 @@ fontstyle = {
     'fs-normal': 'fs-normal',
     'fs-italic': 'fs-italic'
 }
-
 if cssmodules == true
     cssModulesUtils(fontstyle, breakpoints)
 else
     nativeUtils(fontstyle, breakpoints)
+
+fw-normal()
+    font-weight normal
+
+fw-bold()
+    font-weight bold
+
+fw-light()
+    font-weight lighter
```

Comment:
What is the use case fo this? As far as I know we don't use that font-weight. We also don't load that font weight from lato iirc, so it would rely on native interpolation  which is a bit meh.